frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-add-options.feature

```diff
@@ -3,10 +3,10 @@ Feature: Create Pipeline from Add Options
               As a user, I want to create, edit, delete and view the pipeline
 
         Background:
-            Given user has created or selected namespace "aut-pipelines-add-options"
+            Given user has created or selected namespace "aut-pipelines"
```

Comment:
Why have you changed the name of the namespace? Is there an issue due to it

Reply:
Earlier I was facing some issues with the namespace. Will change the namespace back to the previous one